h/auth/policy.py

```diff
@@ -6,6 +6,8 @@
 from pyramid.authentication import CallbackAuthenticationPolicy
 from zope import interface
 
+from h.auth import util
```

Comment:
Moving `check` into the `AuthClientPolicy` class means I have multiple dependencies on the util functions it invokes. I hope this is OK?